### PR TITLE
Deleted old QSG files, add 2 new OSG files, updated go reconciler

### PR DESCRIPTION
**Pull request by ritz303**
## Description
Consolidated multiple Quick Start Guides (QSG) into two QSGs, one for each workflow (Admin, Dev).
These updated QSGs cover the doc requests being tracked under this Jira epic - https://issues.redhat.com/browse/DBAAS-206

## Verification Steps
Add the QSGs to a shared OCP 4.9 cluster in the QuickLab environment.
Direct message me if you want access to this QuickLab environment to review what these QSGs look like.

## Type of change
- [ ] New doc updates (non-breaking change which adds functionality)

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA or in issue number have been completed
- [ ] Unit tests added that prove the fix is effective or the feature works 
- [ ] Documentation added for the feature
- [ ] all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

**Review by jeremyary — CHANGES_REQUESTED** [review]

File: controllers/reconcilers/quickstart_installation/quickstart_installation_reconciler.go [hunk 1]
```diff
@@ -17,15 +17,15 @@ import (
 	"github.com/RHEcosystemAppEng/dbaas-operator/controllers/reconcilers"
 )
 
-//go:embed creating-a-provider-account-instance-quick-start.yaml
-var providerAccountQuickStart []byte
+// 2.11.22 - gunnage : go:embed getting-started-with-openshift-database-access-for-administrators-quick-start.yaml
```
[the remark this answers is not in the document]
Comment (ritz303) [reply controllers/reconcilers/quickstart_installation/quickstart_installation_reconciler.go]: Good to know.  I didn't know that had a special meaning, I thought it was just a comment line.  Thanks for taking a look!

**Review by jeremyary — CHANGES_REQUESTED** [review]
https://github.com/ritz303/dbaas-operator/pull/1

Hey Aron, I missed a formatting recommendation in my previous review. This ^ is a PR that should fix that in your branch.

**Comment by ritz303** [comment]
@jeremyary : Thanks for taking a look!  I've merged your PR into my branch.

**Comment by jeremyary** [comment]
@ritz303 all green, merged up. Thanks for the contribution!

**Pull request merged**